pyshelf/search/sort_base.py

```diff
@@ -0,0 +1,17 @@
+class SortBase(object):
+    @classmethod
+    def get_sort_alias(cls, flag, aliases):
+        """
+            Determines if passed in flag is a SortFlag or an alias of a SortFlag.
+            Args:
+                flag(string): potential sort base alias.
+                aliases(Dict{list}): aliases for attributes.
+
+            Returns:
+                SortBase or None: returns None if invalid flag.
+        """
+        for key, val in aliases.iteritems():
+            if flag.upper() in val:
```

Comment:
I agree the base class makes it hard to understand. eee592a04941dc369bb47c0b8e6e8c0c35f431d4